pymongo/helpers.py

```diff
@@ -88,7 +90,12 @@ def _index_list(key_or_list, direction=None):
             return list(key_or_list)
         elif not isinstance(key_or_list, (list, tuple)):
             raise TypeError("if no direction is specified, key_or_list must be an instance of list")
-        return key_or_list
+        values = []
```

Comment:
Should this handle the ItemsView case too? EG an ItemsView with a mix of string and tuples should be normalized to all tuples.

Reply:
I don't follow, you mean if the value were to be defined as a dictionary and the key was given as `.items()`?

Reply:
Yes, see above on line 89.

Reply:
Done

Reply:
Your change made me realize my mistake. abc.ItemsView is not supported as an element inside key_or_list. We actually don't need to do anything special here since ItemsView always yields (key, value) tuples. Sorry for the confusion. You should remove the ItemsView line you added.

Reply:
I added a test and update the type to allow for that option